python/llm/dev/benchmark/harness/make_table_results.py

```diff
@@ -34,25 +34,30 @@ def make_table(result_dict):
     """Generate table of results."""
     md_writer = MarkdownTableWriter()
     latex_writer = LatexTableWriter()
-    md_writer.headers = ["Model", "Precision", "Task", "Metric", "Value"]
-    latex_writer.headers = ["Model", "Precision", "Task", "Metric", "Value"]
+    md_writer.headers = ["Model", "Precision", "Arc", "Hellaswag", "MMLU", "TruthfulQA","Winogrande", "GSM8K"]
+    latex_writer.headers = ["Model", "Precision", "Arc", "Hellaswag", "MMLU", "TruthfulQA","Winogrande", "GSM8K"]
 
+    tasks = ["arc", "hellaswag", "mmlu", "truthfulqa", "winogrande", "gsm8k"]
```

Comment:
Can we avoid hard code the tasks here? (this file should be a general post processing file?)
e.g. Can we get the tasks from prec_result.keys()?

Reply:
It's more convenient to copy and paste the results to excel... I am trying to avoid further effort on recording the results.